src/main.rs

```diff
@@ -979,6 +980,27 @@ fn file_is_overwritable(path: &Path) -> Result<bool, String> {
         .take_while(|line| line.starts_with("#"))
         .any(|line| line.contains("Autogenerated")))
 }
+#[test]
+fn test_cleanup_autogenerated_completions_in_directory() {
+    let bad_string = String::from(".");
+    let bad_path = Path::new(&bad_string);
+    // need to create a directory path with files for testing
+    //cleanup_autogenerated_completions_in_directory(bad_path);
```

Comment:
Maybe we can discuss strategies for testing+cleanup at the meetup? Could also move `cleanup_autogenerated_completions_in_directory` to a new PR since I think the rest of this branch looks good to merge.

Reply:
yeah, this was meant for a new PR....oops

Reply:
removed